packages/lib/src/components/core/index.ts

```diff
@@ -1,5 +1,6 @@
 // Chart anatomy
 export { default as LumeAxis } from './lume-axis';
+export * from './lume-axis/types';
```

Comment:
Why is it the only one exporting types here?

Reply:
As of now, `lume-axis` is the only component that has types defined within the component folder.

All the other types are shared and defined in `@/types`. So had to take care of only this one for now!
Maybe in the future we may end up having more types defined at component level.

Since the consumer can use Lume axis component, its better to export the types of Lume axis also at `@adyen/lume` :)